### ISSUE TITLE
Possible Windows-only error on save() to PNG

### ISSUE THREAD

**Issue by asarmy** [altair-viz/altair_saver#121]
Getting an error using Altair saver when I am trying to save a chart to a PNG in Jupyter Notebook: `ValueError: Unsupported format: 'png'`

Answer [here](https://stackoverflow.com/questions/62601904/altair-saver-valueerror-unsupported-format-png) on Stack Overflow suggested filing a bug report.
![image](https://user-images.githubusercontent.com/20426220/86830694-dc0a5900-c04a-11ea-8227-4f14991da95f.png)

### System info:
Windows 10
conda 4.8.2
Python 3.8.3
altair 4.1.0 py_1 conda-forge
altair_saver 0.1.0 py_0 conda-forge
vega 3.4.0 py38h32f6830_0 conda-forge
selenium 3.141.0 py38h9de7a3e_1001 conda-forge

### Example Code:
```
import pandas as pd
import altair as alt
from altair_saver import save
alt.renderers.enable('default'); # if in jupyter, ; to suppress output
alt.renderers.enable('altair_saver', fmts=['vega-lite', 'png']);

mytaskbars = pd.DataFrame([
    {"task": "Task1a", "start": '2020-06-01', "end": '2020-09-30', "color": 'royalblue'},
    {"task": "Task1b", "start": '2020-06-01', "end": '2021-03-31', "color": 'deepskyblue'},
    {"task": "Task2", "start": '2020-06-01', "end": '2021-03-31', "color": 'red'},
    ])
    
mytaskbars["start"] = pd.to_datetime(mytaskbars["start"])
mytaskbars["end"] = pd.to_datetime(mytaskbars["end"])
    
    
chart = alt.Chart(mytaskbars).mark_bar(opacity=0.7).encode(
    x=alt.X('start', axis=alt.Axis(title='Date', labelAngle=-45, format = ("%b %Y"))),
     x2 = 'end',
    y=alt.Y('task', axis=alt.Axis(title=None)),
    color = alt.Color('color:N', scale = None)
    )
    
save(chart, "chart_202006.png")
chart
```

### Error Message:
```
ValueError                                Traceback (most recent call last)
<ipython-input-3-13a284c2aca9> in <module>
     19     )
     20 
---> 21 save(chart, "chart_202006.png")
     22 chart

~\anaconda3\envs\geospat_env\lib\site-packages\altair_saver\_core.py in save(chart, fp, fmt, mode, method, **kwargs)
     60     """
     61     if method is None:
---> 62         Saver = _get_saver_for_format(fmt=fmt, fp=fp)
     63     elif isinstance(method, type):
     64         Saver = method

~\anaconda3\envs\geospat_env\lib\site-packages\altair_saver\_core.py in _get_saver_for_format(fmt, fp)
     28         if fmt in s.valid_formats and s.enabled():
     29             return s
---> 30     raise ValueError(f"Unsupported format: {fmt!r}")
     31 
     32 

ValueError: Unsupported format: 'png'
```

**Comment by joelostblom** [comment]
I moved this issue into the altair-saver repo since altair now mainly recommends using vl-convert for saving static chart images.

**Comment by joelostblom** [comment]
Since Altair 5.2, the functionality of Altair Saver is now available in Altair via the [vl-convert](https://github.com/vega/vl-convert) package. Most of the functionality has been available since 5.0, and [the main addition in 5.2 was PDF export](https://github.com/altair-viz/altair/pull/3244). See [the docs on how to save charts](https://altair-viz.github.io/user_guide/saving_charts.html#png-svg-and-pdf-format) for more details

We are going to archive this repo, so I'm closing all the open issues and PRs before doing so. Try out the new options for saving charts mentioned above and if you run into issues, please open an issue directly in the altair or vl-convert repo.
